data/kernels/blendop.cl

```diff
@@ -1017,10 +1017,10 @@ blendop_mask_tone_curve(__read_only image2d_t mask_in, __write_only image2d_t ma
   const int x = get_global_id(0);
   const int y = get_global_id(1);
 
-  if ( x>= width || y >= height || gopacity <= 1e-4f) return;
+  if (x >= width || y >= height) return;
 
   float opacity = read_imagef(mask_in, sampleri, (int2)(x, y)).x;
-  float scaled_opacity = (2.f * opacity - 1.f) / gopacity;
+  float scaled_opacity = (2.f * opacity / gopacity - 1.f);
```

Comment:
So we can have a negative scaled_opacity ?

Reply:
Looking add the line below, it seems that yes. Strange but just want to be sure.

Reply:
> So we can have a negative scaled_opacity ?

Don't really understand that code. This PR is just about making CPU == GPU.